### ISSUE TITLE
Follow-up on recent processing results

### ISSUE THREAD

**Issue by Mathilde-16** [sct-pipeline/STOP-MS-data#6]
As our cohort is not ready yet (sorry for the delay), I've launched the last version of the [script](https://github.com/sct-pipeline/STOP-MS-data/blob/e06ff0be94020b708a9c6ad90be355c96e7a4b43/scripts/process_data.sh) on 72 sessions (12 subjects and each with 6 sessions) I've since the beginning. 

I can't send you directly here the QC zip file due to its size, therefore I share you a WeTransfer [link](https://wetransfer.com/downloads/642713bd46faead2cedfa31f859f96f820240725160954/b703ea14af9930f1e6282207415e49ca20240725161126/988384?trk=TRN_TDL_01&utm_campaign=TRN_TDL_01&utm_medium=email&utm_source=sendgrid) so that you can have access to the QC zip file.

According to this QC, the PMJ seems to be detected with success. 
Regarding the `sct_process_segmentation`, I've some doubts. For example, for _sub-MS03_ses-aera01_ session below, it's C3-C4 which is identified (in red) instead of C2-C3 :

<img width="810" alt="Screenshot " src="https://github.com/user-attachments/assets/b7bf119f-61eb-4311-8638-8ee834d224ce">

Here is output CSA : 
[csa-SC_T1w.csv](https://github.com/user-attachments/files/16381472/csa-SC_T1w.csv)

**Comment by jcohenadad** [comment]
> For example, for sub-MS03_ses-aera01 session below, it's C3-C4 which is identified (in red) instead of C2-C3 :

It is important to understand that the 64mm distance from the PMJ corresponds to the C2-C3 disc in _average_ across a population. The notion of "average" means that it is not _exactly_ 64mm for each individual, but that there is some variation, as illustrated in Figure 3 of the [Bedard et al. paper](https://www.frontiersin.org/journals/neuroimaging/articles/10.3389/fnimg.2022.1031253/full). 

![image](https://github.com/user-attachments/assets/eb74fee5-7d4d-41cf-bb04-118f51f05102)

**Comment by Mathilde-16** [comment]
Hello Julien, 
I've proceeded our data by running the [current script version](https://github.com/sct-pipeline/STOP-MS-data/blob/af7d83c83a1e6a0c3f9bdc44fc34767439565b04/scripts/process_data.sh). Our cohorte is divided into 6 folders corresponding to axial/coronal/sagittal acquisitions, with and without contrast agent, following this naming : _t1_ax, t1_ax_ce, t1_cor, t1_cor_ce, t1_sag, t1_sag_ce_

- As QC results are heavy folders, I’ve created the following WeTransfer link where you can download **QC results** for each scan type : https://we.tl/t-aG1fywojz4 
There isn't qc results for the _t1_ax_ data folder due to it's the largest (about 3500 subjects), consequently, its qc results folder is too big and sharing has failed (even when I create a link containing only qc results for the _t1_ax_ data folder). 

- Please find below a zip folder within **output CSA** files :
[output_CSA.zip](https://github.com/user-attachments/files/16427722/output_CSA.zip)
 
Thank you for your help !

**Comment by Mathilde-16** [comment]
Hi @jcohenadad, thanks to the manual correction, outliers values of CSA have almost all been solved. I'll share you these new results thanks to the One Drive link as soon as possible, I haven't finished working on it yet, but soon !

But I've a trouble with the processing of the _t1_ax_ce_ folder. I've realised that a lot of session results are missing in the CSA output file. Indeed, this file contains results of 9 sessions but the _t1_ax_ce_ folder corresponds to 439 sessions, therefore the `sct_process_segmentation` command hasn't produced any results for most acquisitions, even when the segmentation and the PMJ detection have passed the QC. Then, I've looked at the log files and the error is as below : 

<img width="1468" alt="Screenshot 2024-08-06 at 20 48 45" src="https://github.com/user-attachments/assets/eeee81e7-59b5-4012-84b3-75dc10636f31">

**Comment by jcohenadad** [comment]
pls show sag view of 2-3 subjects where you see this error

**Comment by Mathilde-16** [comment]
Yes sure. I'm going to send you them by email in order to avoid letting several data here in a public repo.
